frontend/main-app/src/components/FormViewer/FormViewerUtils.tsx

```diff
@@ -0,0 +1,86 @@
+import { z, ZodNumber, ZodOptional, ZodString } from "zod";
+
+export const cleanFormData = (formData: FormData) => {
+  const obj = Object.fromEntries(formData);
+  const newObj = {};
+
+  for (const key in obj) {
+    const value = obj[key];
+    if (value === "") {
+      newObj[key] = undefined;
+    } else {
+      newObj[key] = value;
+    }
+  }
```

Comment:
```suggestion
for (const key in obj) {
  const value = obj[key];
  newObj[key] = value === "" ? undefined : value;
}
```
Nothing wrong with your code here, I just feel it makes it more compact when using a ternary operator inside this for loop.

Reply:
Agreed 🙂